js/blocks/NumberBlocks.js

```diff
@@ -789,19 +789,27 @@ function setupNumberBlocks() {
         }
 
         arg(logo, turtle, blk, receivedArg) {
+            let cblk0 = logo.blocks.blockList[blk].connections[0];
             let cblk1 = logo.blocks.blockList[blk].connections[1];
             let cblk2 = logo.blocks.blockList[blk].connections[2];
-
+            
             if (cblk1 === null || cblk2 === null) {
                 logo.errorMsg(NOINPUTERRORMSG, blk);
                 return 0;
             }
 
             let a = logo.parseArg(logo, turtle, cblk1, blk, receivedArg);
             let b = logo.parseArg(logo, turtle, cblk2, blk, receivedArg);
+            let octave = null;
```

Comment:
I think initialising `octave` with `undefined` is better because arguments not passed during function call are assigned `undefined`.

Reply:
+1

Reply:
Done.